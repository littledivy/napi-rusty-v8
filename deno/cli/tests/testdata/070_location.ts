console.log(Location);
console.log(Location.prototype);
console.log(location);
try {
  location.hostname = "bar";
} catch (error) {
  if (error instanceof Error) {
    console.log(error.toString());
  }
}
