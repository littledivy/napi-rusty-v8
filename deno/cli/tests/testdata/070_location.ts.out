[WILDCARD][Function: Location]
Location {}
Location {
  hash: "#bat",
  host: "foo",
  hostname: "foo",
  href: "https://foo/bar?baz#bat",
  origin: "https://foo",
  pathname: "/bar",
  port: "",
  protocol: "https:",
  search: "?baz"
}
NotSupportedError: Cannot set "location.hostname".
[WILDCARD]
